README.md

```diff
@@ -50,6 +50,11 @@ Set up a local Knative cluster using [KinD](https://kind.sigs.k8s.io/):
 ``` bash
 kn quickstart kind
 ```
+For [KinD](https://kind.sigs.k8s.io/), a local registry will no longer be created by default. Users will need to pass the `--registry` flag if they wish to create a local registry:
+
+```bash
+kn quickstart kind --registry
+```
 
```

Comment:
In light of the changes made in #468 , it would also be good to add a note about tag resolution:

```suggestion

Note: we automatically configure tag resolution for the local registry when this flag is passed.
```